### PR TITLE
Update & Refactor participants/show view.

### PR DESCRIPTION
**Pull request by dinocosta**

### CODE REVIEW

File: app/views/participants/_participant_badges.slim [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- if @participant.badges.empty?
+  p
+    | #{@participant.full_name} ainda não conquistou nenhum #{t('badge')}.
+- else
+  - @participant.badges.each do |badge|
+    .col-md-3.col-sm-3.col-xs-6
```
Comment (frm) [inline app/views/participants/_participant_badges.slim:6]: Extract these into a `app/views/participants/_badge.slim` partial?

File: app/views/participants/_participant_info.slim [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    | Username
+  li.participant-info-item
+    = @participant.username
+
+  li.participant-info-category
+    | Sexo
+  li.participant-info-item
+    = @participant.sex_to_s
+
+  - if @participant.birthday != nil
```
Comment (frm) [inline app/views/participants/_participant_info.slim]: Redundant `!= nil`, just `if @participant.birthday`.

File: app/views/participants/_participant_info.slim [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - if @participant.minho_univ_student_id
+      li.participant-info-category
+        | Nº de Aluno UMinho
+      li.participant-info-item
+        = @participant.minho_univ_student_id
+
+    li.participant-info-category
+      | Curso
+    li.participant-info-item
+      - if @participant.is_inf_eng_student_at_minho_univ
```
Comment (frm) [inline app/views/participants/_participant_info.slim:40]: wat. How about a `@participant.uminho_student(:informatica)`?

**Review by frm — CHANGES_REQUESTED** [review]

File: app/views/participants/_participant_badges.slim (hunk 1 above)
Comment (dinocosta) [reply app/views/participants/_participant_badges.slim:6]: Sure :+1:

File: app/views/participants/_participant_info.slim (hunk 2 above)
Comment (dinocosta) [reply app/views/participants/_participant_info.slim]: Sorry, I copy pasted from another file 😬

File: app/views/participants/_participant_info.slim (hunk 3 above)
Comment (dinocosta) [reply app/views/participants/_participant_info.slim:40]: @frmendes Exactly my reaction when I saw it for the first time haha
I agree that it is better to create a method on the Participant model as you recommended. Will get on with it ;)

File: app/models/user.rb [hunk 4]
```diff
@@ -114,6 +114,10 @@ def self.reset_badge_counters(user)
     User.reset_counters user.id, :badge_acquisitions
   end
 
+  def uminho_student(course)
```
Comment (dinocosta) [inline app/models/user.rb]: @frmendes Is this it? I was trying to wrap my head around if the argument was really needed...

**Comment by dinocosta** [comment]
@frmendes Is there anything else I should change or can I close this PR? :)

**Pull request merged**